extensions/wikia/EditPageLayout/css/core/wide.scss

```diff
@@ -263,6 +241,35 @@
 	}
 }
 
+#EditPage {
+	&.toolbar-expanded {
+		.preview_box {
+			border: 0;
+			clear: both;
+
+			h3 {
+				margin-top: 8px;
+			}
+
+			.preview_mobile {
+				padding-top: 3px;
+			}
+
+			.preview_desktop {
```

Comment:
don't you think we can simplify these nested selectors
#EditPage.toolbar-expanded .preview_box .preview_mobile
by moving preview_mobile, preview_desktop and h3 level up, and possibly adding a class to h3?

up to you.